database/migrations/2020_10_27_011703_create_exercise_set.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateExerciseSet extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('exercise_set', function (Blueprint $table) {
+            $table->id();
+            $table->integer('workoutlog_id');
+            $table->integer('exercise_id');
+            $table->timestamps();
+            $table->boolean('complete')->default(true);
```

Comment:
I think this complete should be false for the default value
```suggestion
            $table->boolean('complete')->default(false);
```